src/Symfony/Component/Form/Extension/Core/Type/TimezoneType.php

```diff
@@ -46,6 +46,7 @@ public function configureOptions(OptionsResolver $resolver)
             'choice_translation_domain' => false,
             'input' => 'string',
             'regions' => \DateTimeZone::ALL,
+            'invalid_message' => 'The timezone is invalid.',
```

Comment:
`time zone`

Reply:
I've updated the message, although the `Type` is called `TimezoneType` which may be better called `TimeZoneType`